src/commands/es/snapshots/restore.ts

```diff
@@ -0,0 +1,65 @@
+import { flags } from '@oclif/command'
+import { Client } from '@elastic/elasticsearch'
+
+import { Kommand } from '../../../common'
+
+export default class ESRestore extends Kommand {
+  static initSdk = false
+
+  static description = 'Restore a snapshot into an ES instance'
+
+  static flags = {
+    host: flags.string({
+      char: 'h',
+      description: 'Elasticsearch server host',
+      default: process.env.KUZZLE_HOST || 'localhost',
+    }),
+    port: flags.string({
+      char: 'p',
+      description: 'Elasticsearch server port',
+      default: process.env.KUZZLE_PORT || '9200',
+    }),
+    help: flags.help(),
+  }
+
+  static args = [
+    { name: 'repository', description: 'ES repository name', required: true },
+    { name: 'name', description: 'ES snapshot name', required: true },
+  ]
+
+  async getIndices(esClient: Client) {
+    const beforeResponse = await esClient.cat.indices({ format: 'json' })
+
+    const indices: string[] = beforeResponse.body
```

Comment:
Type is automatically inferred here

```suggestion
    const indices = beforeResponse.body
```